document-portal/document-portal-fuse.c

```diff
@@ -2010,7 +2012,7 @@ xdp_fuse_read (fuse_req_t req,
   buf.buf[0].fd = file->fd;
   buf.buf[0].pos = off;
 
-  fuse_reply_data (req, &buf, FUSE_BUF_SPLICE_MOVE);
```

Comment:
From the docs it sounds like `FUSE_BUF_SPLICE_MOVE` should be able to fall back to not using splice. Why is this needed?

Reply:
From what I understand, the docs mean that a splice move **may** decay to copying in some circumstances, but it still requires that the splice feature is supported by the FUSE server.